server/views/pages/calendar-details.ejs

```diff
@@ -13,17 +13,31 @@
 
   <% include ../partials/beta-banner %>
 
-  <a href="/calendar/<%=moment(date).format('YYYY-MM-01')%>" class="govuk-back-link">Back</a>
+    <main id="main-content" role="main">
+        <div class="tabs-wrapper">
+            <ul class="tabs">
+                <li><a class="active" href="/calendar/<%=startDate.format('YYYY-MM-01')%>">Calendar</a></li>
+                <li>
+                    <a href="/notifications">Notifications
+                    </a>
+                </li>
+            </ul>
+        </div>
 
-  <main id="main-content" class="govuk-main-wrapper govuk-!-margin-bottom-8" role="main">
+        <div class="calendar-controls center-align">
+            <h1 class="govuk-heading-m govuk-!-margin-bottom-0"><%= startDate.format('dddd, Do MMMM YYYY') %></h1>
+            <a id="btnPreviousShiftDetail" href="/details/<%= startDate.clone().subtract(1,'d').format('YYYY-MM-DD') %>"
```

Comment:
Personally I'm not a fan having logic in templates. I feel like html is already full of so much html faff that it's not clear what's going on.

Refactoring this out to helpers or passed config would be clearer. This is a personal preference/stylistic kinda thing tho, so not blocking.

Reply:
I agree there is work here, but i'd rather put the effort into nunjucksing it at the same time..